about_regular_expressions.exs

```diff
@@ -0,0 +1,41 @@
+defmodule AboutRegularExpressions do
+  use Koans
+
+  think "A simple regular expression" do
+    regex = ~r/This [W|w]orks!/
+
+    completion = __?
+
+    assert ("This W" <> completion) =~ regex
+    assert ("This w" <> completion) =~ regex
+  end
+
+  think "Regular expressions can use options" do
+    regex = ~r/this works!/i
+
+    assert_? "this works!" =~ regex
+    assert_? "ThIs wOrKs!" =~ regex
+  end
+
+  think "Regular expressions don't have to be created with the sigil (though ~r/.../ is more idiomatic)" do
+    regex = Regex.compile!(__?)
+
+    assert regex == ~r/this works!/
```

Comment:
oh, right, it's comparing the expressions as values... maybe actually use the regex instead?